src/components/TabsBar/TabsBar.vue

```diff
@@ -0,0 +1,216 @@
+<template>
+    <div id="tabsBar" class="noSelect pointerCursor">
+        <draggable
+            :key="updateCount"
+            v-model="SimulatorState.circuit_list"
+            class="list-group"
+            tag="transition-group"
+            :component-data="{
+                tag: 'div',
+                type: 'transition-group',
+                name: !drag ? 'flip-list' : null,
+            }"
+            v-bind="dragOptions"
+            @start="drag = true"
+            @end="drag = false"
+        >
+            <template #item="{ element }">
+                <div
+                    :id="element.id"
+                    :key="element.id"
+                    style=""
+                    class="circuits toolbarButton"
+                    :class="tabsbarClasses(element.id)"
+                    draggable="true"
+                    @click="switchCircuit(element.id)"
+                >
+                    <span class="circuitName noSelect">
+                        {{ truncateString(scopeList[element.id].name, 18) }}
+                    </span>
+                    <span
+                        id="scope.id"
+                        class="tabsCloseButton"
+                        @click="closeCircuit($event, element)"
+                    >
+                        x
+                    </span>
+                </div>
+            </template>
+        </draggable>
+        <button @click="createNewCircuit()">&#43;</button>
+    </div>
+    <MessageBox
+        v-model="SimulatorState.dialogBox.create_circuit"
```

Comment:
I'm not able to find the MessageBox component accepting v-model, can you add some context here?